### ISSUE TITLE
Forward rules are not getting added back after flushing manually

### ISSUE THREAD

**Issue by bazuchan** [cloudnativelabs/kube-router#588]
kube-router add folowing iptables rules:
```
-A FORWARD -i kube-bridge -m comment --comment "allow outbound traffic from pods" -j ACCEPT
-A FORWARD -o kube-bridge -m comment --comment "allow inbound traffic to pods" -j ACCEPT
-A FORWARD -o v4 -m comment --comment "allow outbound node port traffic on node interface with which node ip is associated" -j ACCEPT
```
If I flush iptables rules manually `iptables -F`, all other rules gets added back, bot not this ones.

**Comment by eric** [comment]
I've run into this as well — changes made to firewal rules via `firewall-cmd` will remove the kube-router rules and they do not get re-added.

**Comment by bazuchan** [comment]
I have fixed it for myself, will make PR today or tomorrow.

**Comment by roffe** [comment]
Fixed by #603 thank you @bazuchan
